### PR TITLE
Login service bugfixes and documentation

### PR DESCRIPTION
**Pull request by ModellbahnFreak**
This branch fixes many small bugs on the login service.

It also provides the new debug page that allows for easier execution of login service queries and documentation to go along with it and explain how to use it.

As it introduces database migrations on the production deployment, it requires https://github.com/ccims/gropius/pull/8 to be merged as dependency of this

### CODE REVIEW

File: core/src/main/kotlin/gropius/service/user/GropiusUserService.kt [hunk 1]
```diff
@@ -109,4 +110,11 @@ class GropiusUserService(
         }
     }
 
+    /**
+     * Find the ids of all [GropiusUser]s
+     */
+    suspend fun findGropiusUserIds(): List<ID> {
+        //return repository.findAll().map { it.graphQLId }.collectList().awaitSingle()
```
Comment (nk-coding) [inline core/src/main/kotlin/gropius/service/user/GropiusUserService.kt:117]: commented out code

File: login-service/src/migrationDataSource.config.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { DataSource } from "typeorm";
+import { randomBytes } from "crypto";
+
+function setEnvVariables() {
+    //process.env.NODE_ENV = "production";
```
Comment (nk-coding) [inline login-service/src/migrationDataSource.config.ts:5]: does this still serve some purpose?

File: login-service/static/ui/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            white-space: nowrap;
+        }
+    </style>
+    <script src="/vue/dist/vue.global.prod.js"></script>
+</head>
+
+<body>
+    <main id="app">
+        <h1>Gropius Login UI</h1>
+        <i>This is WIP. Not functioning yet</i>
```
Comment (nk-coding) [inline login-service/static/ui/index.html:31]: ?

File: core/src/main/kotlin/gropius/service/user/GropiusUserService.kt (hunk 1 above)
Comment (ModellbahnFreak) [reply core/src/main/kotlin/gropius/service/user/GropiusUserService.kt]: Fixed in cb763a5

File: login-service/src/migrationDataSource.config.ts (hunk 2 above)
Comment (ModellbahnFreak) [reply login-service/src/migrationDataSource.config.ts]: Removed in cb763a5

**Review by nk-coding — APPROVED** [review]

**Pull request merged**